src/bill/mod.rs

```diff
@@ -19,12 +19,15 @@ pub fn get_path_for_bill_keys(key_name: &str) -> PathBuf {
     path
 }
 
-pub fn bill_from_byte_array(bill: &[u8]) -> BitcreditBill {
-    BitcreditBill::try_from_slice(bill).unwrap()
+pub fn bill_from_byte_array(bill: &[u8]) -> Result<BitcreditBill, borsh::maybestd::io::Error> {
```

Comment:
This doesn't compile.

Generally, since the changed methods in this file are mostly used in the `blockchain` module, it would make sense to leave them as they are as well (since they will be deleted soon)